charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```
@@ -12,7 +12,118 @@ spec:
     spec:
       containers:
         - name: repo-creator
-          image: quay.io/redhat-ai-dev/helm-chart-application-gitops:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #!/usr/bin/env bash
+              set -o nounset
+              set -o pipefail
+
+              setup_data() {
+                 gh repo clone "$GITHUB_SOURCE_REPO" app-source -- --no-tags --branch="$GITHUB_DEFAULT_BRANCH" --single-branch
+                 gh repo clone "$GITHUB_TETKON_SOURCE_REPO" tekton-source -- --no-tags --single-branch
+                 gh repo clone "$GH_REPO" "$APP_NAME" -- --no-tags --branch="$GITHUB_DEFAULT_BRANCH" --single-branch
```

I think we can use `--depth` here also, see https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/25/files#r1853115886
```suggestion
                 gh repo clone "$GITHUB_SOURCE_REPO" app-source -- --no-tags --branch="$GITHUB_DEFAULT_BRANCH" --depth 1
                 gh repo clone "$GITHUB_TETKON_SOURCE_REPO" tekton-source -- --no-tags --depth 1
                 gh repo clone "$GH_REPO" "$APP_NAME" -- --no-tags --branch="$GITHUB_DEFAULT_BRANCH" --depth 1
```

---

part of next push